Makefile

```diff
@@ -479,7 +480,8 @@ $(KVM_BUILD_IMAGE): installers/linux/kvm/Dockerfile
 	@echo "$(@) successfully built"
 
 kvm_in_docker:
-	docker inspect -f '{{.Id}} {{.RepoTags}}' $(KVM_BUILD_IMAGE) || $(MAKE) $(KVM_BUILD_IMAGE)
+	docker inspect -f '{{.Id}} {{.RepoTags}}' $(KVM_BUILD_IMAGE) || \
```

Comment:
```
docker inspect -f '{{.Id}} {{.RepoTags}}' $(KVM_BUILD_IMAGE) ||  docker pull $(KVM_BUILD_IMAGE) || $(MAKE) $(KVM_BUILD_IMAGE)
```

As we are doing it in two places, maybe worth extracting a function?

Reply:
Normally live by the "one, two, many (1,2,N)" rule